examples-flakes/1-hello-world/flake.nix

```diff
@@ -0,0 +1,44 @@
+{
+  inputs = {
+    nixpkgs-mozilla = { url = github:mozilla/nixpkgs-mozilla; flake = false; };
+    cargo2nix.url = github:onsails/cargo2nix/flake;
```

Comment:
note it's my fork here for now to let CI pass

Reply:
Regarding the fork reference for CI, can we not use the local source as a flake input? I don't want to have to bump flake inputs after merges. Currently, the overlay version in use to test build is from the repo ref being tested, and referencing it via external URL seems to break that

Reply:
unfortunately, flakes don't support relative input paths [yet](https://github.com/NixOS/nix/issues/3978)

Reply:
latest nix supports relative paths.
I've force-pushed changes without reference to the fork.